### ISSUE TITLE
Laravel 5.1 : JSON property "timespent" must not be NULL 

### ISSUE THREAD

**Issue by zandotproperty** [lesstif/php-jira-rest-client#102]
Hi,
I use this library with laravel 5.1, and got the following error:
Using version 1.5.*
```
JsonMapper_Exception in JsonMapper.php line 183:
JSON property "timespent" must not be NULL
in JsonMapper.php line 183
at JsonMapper->map(object(stdClass), object(IssueField)) in JsonMapper.php line 258
at JsonMapper->map(object(stdClass), object(Issue)) in IssueService.php line 23
```

**Comment by lesstif** [comment]
hi @zandotproperty.

this bug was already reported #100, and fixed 1.14.2.

Running *composer update* will fix your problem.

thanks.

**Comment by zandotproperty** [comment]
hi @lesstif 

May be problem with my laravel 5.1 not support dotenv v2.*  when update to v1.14.2 that require dotenv 2.*

thanks.

**Comment by lesstif** [comment]
hi @zandotproperty 

I didn't know Laravel was using dotenv v1.1.

So I have plan to modify this library to support both 1.1 and 2.x dotenv.

thanks.

**Comment by lesstif** [comment]
hi @zandotproperty 

i was released fix version [1.14.4](https://github.com/lesstif/php-jira-rest-client/releases/tag/1.14.4).

thanks
